src/main/java/org/openstreetmap/atlas/utilities/maps/LongToLongMultiMap.java

```diff
@@ -74,6 +91,86 @@ public void add(final Long key, final long value)
         this.put(key, result);
     }
 
+    @Override
+    public boolean equals(final Object other)
+    {
+        if (other instanceof LargeMap)
```

Comment:
Should this be `instanceof LongToLongMultiMap`?

Reply:
Yeah... copy paste strikes again.